test/parallel/test-domain-timers-uncaught-exception.js

```diff
@@ -0,0 +1,14 @@
+const domain = require('domain').create();
+var first = false;
+
+domain.run(function() {
+  setTimeout(function() { throw Error('FAIL'); }, 1);
+  setTimeout(function() { first = true; }, 1);
+  setTimeout(function() {
+    if (first === false) {
+      process.exit(1);
+    }
+  }, 2);
+});
+
+domain.once('error', function() {});
```

Comment:
ahh yes! It should be called because of the unhandled exception falling through.
Will wrap with common.mustCall.